src/firmware/app/control/python_controller/difference_equation_test.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+"""
+This file contains the unit tests for the DifferenceEquation class
+
+"""
+
+import numpy as np
+import control as ct
+from control.matlab import *
+import unittest
+import differenceequation as de
```

Comment:
Just realized this is weird because the file is named `differenceequation.py`. Can we change it to `difference_equation.py` to match our style guide?

Reply:
Renamed 👍